adam/tests/conftest.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
```

Comment:
Where is this file used?

Reply:
pytest framework imports it automagically before running any of the tests in `adam/tests` (and any subdrectories). It's a way to define fixtures and [generally hook into pytests internal processing](https://stackoverflow.com/a/34520971).